### DISCUSSION TITLE
341AB5C9-1C08-445F-A2F4-B8B30326E3E1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#18] [General]
# 341AB5C9-1C08-445F-A2F4-B8B30326E3E1

Kepler-80, also known as KOI-500, is a red dwarf star about 1,218 light-years away in the constellation Cygnus. There are six planets (two terrans, two superterrans and two neptunians) discovered, all orbiting at a space no greater than 0.14 AU from their star

https://celestia.mobi/addon?item=341AB5C9-1C08-445F-A2F4-B8B30326E3E1

**Comment by exocritic5457** [comment]
the innermost planet's texture is flipped backwards, contrary to what the preview image wants me to believe.

**Reply by exocritic5457** [reply]
Thankfully also aged like milk, no more obnoxious dark lava
